src/patternfly/components/Table/table.scss

```diff
@@ -385,14 +383,22 @@
     transition: var(--pf-c-table__expandable-row--Transition);
   }
 
-  // no padding on standard td
-  td:only-child {
-    padding: 0;
-    padding-left: var(--pf-c-table__expandable-row--before--Width); // set padding-left to adjust for left border.
+  /* stylelint-disable */
+  td {
+    &.pf-m-no-padding {
+      padding: 0;
+      padding-left: var(--pf-c-table__expandable-row--before--Width); // set padding-left to adjust for left border.
```

Comment:
Can you write this as `padding: 0 0 0 var(--pf-c-table__expandable-row--before--Width);`?